include/constants/metatile_labels.h

```diff
@@ -1,6 +1,9 @@
 #ifndef GUARD_METATILE_LABELS_H
```

Comment:
GUARD_CONSTANTS_METATILE_LABELS_H

Reply:
porymap will overwrite this with `GUARD_METATILE_LABELS_H`, which is what pokeruby/pokeemerald use

Reply:
:man_facepalming: